build-osx.sh

```diff
@@ -61,7 +61,7 @@ NC=`tput init`
 
 prerequisite_check()
 {
-    if [ ! -f vst3sdk/LICENSE.txt ]; then
+    if [ ! -f vst3sdk/LICENSE.txt ] || [ ! -f spdlog/LICENSE ]; then
         echo
```

Comment:
This one is here because we had more than one “why doesn’t it build” question from people who didn’t run git submodules so a polite check with a clearer error seemed friendly.